pkg/executor/aggregate/agg_hash_partial_worker.go

```diff
@@ -99,7 +90,7 @@ func (w *HashAggPartialWorker) fetchChunkAndProcess(ctx sessionctx.Context, hasE
 	}
 
 	waitStart := time.Now()
-	ok := w.getChildInput()
+	chk, ok := w.getChildInput()
 	updateWaitTime(w.stats, waitStart)
```

Comment:
I think you should move this line after `defer w.inflightChunkSync.Done()`

Reply:
> I think you should move this line after `defer w.inflightChunkSync.Done()`

done